Java/other/RecursionCount.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class RecursionCount {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter first string:");
+        String str1 = scanner.nextLine();
+        System.out.println("Enter second string:");
+        String str2 = scanner.nextLine();
+        scanner.close();
+
+        System.out.println("Number of occurrences of \"" + str2 + "\" in \"" + str1 + "\":" + countSubstring(str1, str2));
```

Comment:
Add the backslash in Double Quote

Reply:
@siddharth25pandey sir it's not for putting backslash into the output, its escape character to show a double quote in output, I have put a sample of it in the comments below.

Reply:
cool